tests/test_latest.py

```diff
@@ -67,7 +68,7 @@ def default_multiscale():
 
 def test_multiscale(default_multiscale):
     base_schema, strict_schema = fetch_schemas("latest", schema_name="image")
-    jsc.validate({"multiscales": [default_multiscale.dict()]}, strict_schema)
+    jsc.validate({"multiscales": [default_multiscale.model_dump()]}, strict_schema)
```

Comment:
API change: https://docs.pydantic.dev/latest/migration/#changes-to-pydanticbasemodel

`.dict()` still works but throws deprecationwarnings.